functions/HttpTriggerAPIUsers/__init__.py

```diff
@@ -20,21 +21,23 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         conn = get_db_connection()
     except (pyodbc.DatabaseError, pyodbc.InterfaceError) as e:
         logging.critical("Failed to connect to DB: " + e.args[0])
-        logging.debug("Error: " + e.args[1])
+        logging.info("Error: " + e.args[1])
         return func.HttpResponse(status_code=500)
         
     logging.debug("Connection to DB successful!")
 
+    create_users_table(conn)
+
     try:
         # Return results according to the method
         if method == "GET":
             logging.debug("Attempting to retrieve users...")
-            all_users_http_response = get_users(conn)
+            all_users_http_response = get_users(conn, init_redis())
             logging.debug("Users retrieved successfully!")
             return all_users_http_response
 
         elif method == "POST":
-            logging.debug("Attempting to add user...")
+            logging.critical("Attempting to add user...")
             user_req_body = req.get_json()
```

Comment:
Not too important but what was your reasoning for making this a critical log level?

Reply:
Because the only ones I can see are critical & info - I should change this to info actually, thanks for the catch!